packages/react/src/components/TwoColumnPanel/TwoColumnPanel.tsx

```diff
@@ -231,22 +232,24 @@ const TwoColumnPanel = forwardRef<HTMLDivElement, TwoColumnPanelProps>(
         {...props}
         ref={ref}
       >
-        <FocusTrap
-          active={!isCollapsed && isFocusTrap}
-          focusTrapOptions={{
-            escapeDeactivates: true,
-            allowOutsideClick: true,
-            fallbackFocus: columnLeftRef.current as HTMLElement
-          }}
-          containerElements={[columnLeftRef.current as HTMLElement]}
-        />
-        <ClickOutsideListener
-          onClickOutside={handleClickOutside}
-          target={columnLeftRef.current as HTMLElement}
-        />
-        {isCollapsed ? null : skipLink}
-        {showPanel ? ColumnLeftComponent : null}
-        {ColumnRightComponent}
+        <>
```

Comment:
?

Reply:
I no longer have the exact error message, but it was something like...

> `<div>` is expecting a single child, but multiple were provided

This was the easiest/less intrusive fix, as it has 0 effect on anything other than TypeScript.